syntax = "proto3";

package grpc;

message Empty {}

message Data { 
    string name = 1; // Name of the client
    string message = 2; // Message of the client
}

service ChatServer {
    rpc ChatStream (Empty) returns (stream Data);
    rpc SendData (Data) returns (Empty);
}